src/test/java/io/xstefank/wildfly/bot/PRFormatOverrideMessage.java

```diff
@@ -0,0 +1,105 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHCommitState;
+import org.kohsuke.github.GHEvent;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHPullRequestCommitDetail;
+import org.kohsuke.github.GHPullRequestFileDetail;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.PagedSearchIterable;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.mockEmptyComments;
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.processPullRequestMock;
+
+@QuarkusTest
+@GitHubAppTest
+public class PRFormatOverrideMessage {
+
+    private String wildflyConfigFile;
+
+    @BeforeEach
+    void setUp() {
+        wildflyConfigFile = """
+                wildfly:
+                  format:
+                    title:
+                      message: "Custom title message"
+                    commit:
+                      message: "Custom commit message"
+                """;
+    }
+
+    @Test
+    public void testOverridingTitleMessage() throws IOException {
+        String commitMessage = "WFLY-00000 commit";
+        GitHubAppTesting.given()
+                .github(mocks -> {
+                    mocks.configFile(RuntimeConstants.CONFIG_FILE_NAME).fromString(wildflyConfigFile);
+
+                    GHPullRequestCommitDetail mockCommitDetail = Mockito.mock(GHPullRequestCommitDetail.class);
+                    PagedSearchIterable<GHPullRequestCommitDetail> commitDetails = GitHubAppMockito.mockPagedIterable(mockCommitDetail);
+                    Mockito.when(mocks.pullRequest(1352150111).listCommits()).thenReturn(commitDetails);
+
+                    GHPullRequestCommitDetail.Commit mockCommit = Mockito.mock(GHPullRequestCommitDetail.Commit.class);
+                    Mockito.when(mockCommitDetail.getCommit()).thenReturn(mockCommit);
+                    Mockito.when(mockCommit.getMessage()).thenReturn(commitMessage);
+
+                    mockPullRequestContents(mocks.pullRequest(1352150111));
+                })
+                .when().payloadFromClasspath("/pr-fail-checks.json")
+                .event(GHEvent.PULL_REQUEST)
+                .then().github(mocks -> {
+                    GHRepository repo = mocks.repository("xstefank/wildfly");
+                    Mockito.verify(repo).createCommitStatus("860035425072e50c290561191e90edc90254f900",
+                            GHCommitState.ERROR, "", "Failed checks: title", "Format");
+                    Mockito.verify(mocks.pullRequest(1352150111)).comment(PullRequestFormatProcessor.FAILED_FORMAT_COMMENT
+                            .formatted("- Custom title message"));
+                });
+    }
+
+    @Test
+    public void testOverridingCommitMessage() throws IOException {
+        String commitMessage = "commit message";
+        String commitSha = "sha123";
```

Comment:
I think this is already in constants somewhere, also we can use the builder now?